Contrib/NuGet/OpenEXR-msvc-x86-build.nuget.bat

```diff
@@ -55,31 +55,19 @@ rd /s /q %BUILDTREE%\OpenEXR
 mkdir %BUILDTREE%\OpenEXR
 cd %BUILDTREE%\OpenEXR
 cmake -G %cmake_platform% ^
--DBUILD_ILMBASE_STATIC:BOOL=ON ^
+-DBUILD_BUILD_SHARED_LIBS:BOOL=OFF ^
```

Comment:
@meshula, looking at this again, shouldn't this be OPENEXR_BUILD_SHARED_LIBS, or just BUILD_SHARED_LIBS?